docs/notebooks/subset.ipynb

```diff
@@ -0,0 +1,598 @@
0JDMVUPDZMTj3VaLgTxYUK5Jfsr1Jq3R8nOsnPUyjn7Mm9effOL737+uYLzlfsizEsjHAHY3a6Kjo8MIDtT287wmCt+BQMAI1xTCZQwDrQ3S64IpR2XQM12NOAfdktDIttqgPU2KQbtq6/VQDXjyw7zjrF69GhMnTsTcuXMB9DOYixcvxm677WZM0g888AC++tWvwuPxYO7cubjiiiuw7777upZJMyqhN/pKtQV8XjLy8pOdLyemrIf3ccOQAoQsmxOHJlAZhJVMJh0T0Q1uwyUlZx41TrNbc3OziU2YOnWqow62QUrmwWDQMcl54iczJPE5Lf0TOkUq20bB0RZ4Jo9LZ7nyMDepIeB3KdjIumRmJtln+Xwel1++omj/jgWQOcnn845UnYsXnwMAuP76WwAAl1222PQ7x2UkBaGHHrrXjJ30oa+2kLBgwZlWzdHq1f1WSmp0pQDP/1euXG0sCXKOrFr1A5x11rfM2uZcSSaTjgwzmtDrWByeGE5ITRifJyPoZiKWjFAulzNjOWXKFEf5pCty7RSzJkyZMqFk31YDxx57LH70ox9hxowZI1JfpWhvLz8d64UXnmvmshxXarEBGDrN/6XiQyo2qGihe0MkEilwd2EmFFoF5Fyj21EkEnHMSSmIaEyYMMExf1iHFhJ4/Zhjji+7b4YbXV1bzTrgPsE+ZNa6dDoNr9eLCRMG5rZcR/yd40MmT1sdSEei0XFhYSzgiiuWAACam5sdsQjyO4CC63I/4BzhX1t2waxKAAAgAElEQVRbm0PBSYGb60wqZ0OhkFmLMj6ou7sb8XjckfmRnzpWAXAesCbbPhgBwXav/t+mhNYZHjdu3FKq+4vuFSOe3WjJkiWYPXu2w2f15JNPNt8/8YlPYMOGDUWFhHJQiYAA2P39tT+ZbfC0MCE3cr2h8z5tZeB3HRCjpdBS7yQZE2n5oPQs75HByPrd5ALMZrPo6+szk1+mBdP95abpZNt1ALQE2yu1A7IP5YYdCoUcFgfNfFGSL9amsYxK2zpa8QFSQACqLxy4wc1SBRQ/g4S/E3KuMWON23rVdbLPpcCv5yGflxonWaZOEiCf8fl86O3tHVPnUbhh9uzZY1ZAKBcUzG1rSacfJWxCqUwCQVBAjEQiRoHDe8jU6nNl6PJQU1ODhoYGo82UcWsMcuZvdXV1xn/f6/Wip6cHQL/g29PTg/b2dpNFaZdddgEA3HffnQCAuXPPGELvVQctLTPQ0THgQ21zs2KcHZVIEj09PcbKwD5OpVLI5XJGyOOaGxC63FPTjmPkkcvl0Nvba4RAekZoxYqb4MC4y76+Pocyh3wLXfoAOJSydMdLpVJIp9MIhUKmDGY8CofDDiGUdWtIpY5MXFEO3HinSp6TiWLK5YOLYUSFhF/96lfw+Xw455xzzLW3334bt956K2644QZks1msW7cOxxxzzEg2axz/Qli27CKHEDHWLQvnnz/fEICx7Pc/GjjnnO8UTYNaDAsWnFnl1vzrYs8998SiRYtw8MEHO+boiSeOXKaroWDhwrMAwLjHRiIR8xsFPKmNjsVixh1BMy3S3YWum7W1tYhEIgiFQoZxb2xsBDCgeaQwCQwEMgMDVohEIoFgMOhQIqVSKQQCAaMRBeCwyNL9IRKJoK6uDjU1Ndi+fTu6u7sLrPFj5dT5yZM/AKDfqsDgU2mx4f/d3d1GIMrn8wgEAgiHwyY+im5XzG6jma8BP/NxIWEsgOPT29tr1llfX1/BQYNSa6/HlEJlOp1GIpFAIBAwbnyJRAJ+v98ImgAMA8+5IH+jECmVRDx5mdZdnQCH97BcBkJL64NsdzkMfLF7bAoNeX81BARgGN2N1q9fj+uuuw5btmxBbW0tpk2bZnwn6S+5++67Y9myZVi5ciWeffZZeL1eHHnkkTjrrLOKli0JnNRMu/kpSpTqRK3xlpNEuxFxQ5REjJNCurtoy4Ik6JKApVIpkzFFS8qVQObLrq2txaRJk5DP5zFjxgxMnTrVca80i7Euqb0Jh8MmeCeXy6GxsRH19fUFcQ26fv0bJXkG/GgTMF2jbJYWbp7SbYk+uNL0J92RdDyD1jroOSL9jMcCLrjge679yz6QaQ5lX15xxXUj0saHH+7PSDYS2VLOPfe7AAozk+VyOUNPpBaWWLlytWECCd4Xj8cLNDBc03fccS8A4JRT/t2x9qUWiZY2nZVMuoFQu0wtsAy6ZDkybR0ZS2qvm5ubDU2waanHgrvRkiVLrNdXrBgbAribu9E553zHETfAQ5gAd+tBIBBANBo1bqOSjugYF9JSWoXC4bApv6GhAZlMBt3d3airq3NYGGRdZGy2b99u5hUtqUwHGovFEI/H0dzcjIkTJ5qMLowRyWQyRjgJhUKYNm0aotEo5syZg76+PmNVYP3nnnvhIHq5uqCQAAzsJzLrXl9fH5qamhyCAu/t7e1FMBhEPp9HKBQy65z7IDCQBjmXs4/zOEYWl1/eP+dKWW4l9J7OjHNSMCCPQBeigXEfoMmAkw+yZWbkMxQQZBySrb0UUAEUZMGrRDiw3a/5XF6T1zVvWsrlaFTcjfbZZ5+y05pW43ROLSjwu20A9X0SktHQ5hrNvGr3AV2uFiqkMKGFEABGc2LzNS0HrE/64WWzWXzgAx8wpjNKuXqByf+5uLxeL7q7u5FMJs1GBPRrwaR7lWSi5PtqJlfWIzVETNtFxomTXAoiUiiTwdk6Bz3rke4fuj2yTWMV2hee0GlopUZzpN+JOaZHEhT+bEH+pd5fE9wf/vAefPvbpzqIt1vyAB20LGmBdhPkvTKYleNEgcI2F7leqTwAYIRhPZ/H0vwdK8LAYCHdv7xer9FeSrolAxsl/ZbCmz7wjGWzXMn0plIpbN26Fdls1jDzEyZMQDweN/QvFAqhqakJ2WwWjY2N2Lx5wBWHbmjhcNgwwR0dHchkMmhpaTH3RiIRzJo1yzBRPp8P8XgckUgEL7zwAmbPno2tW7eirq7OxDBcc81luPji
```

Comment:
Addind a basemap would require importing cartopy and everything, I think that's out of scope for the example. And this dataset is the only one with 2D lat/lon that we are sure the user can access (since it comes with xarray). I can add some text...

Reply:
Sounds good